tests/xtimer_usleep/tests/01-run.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Francisco Acosta <francisco.acosta@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+from datetime import datetime
+
+class InvalidTimeout(Exception):
+    pass
+
+def testfunc(child):
+    exp_diff1 = 1000000
+    exp_diff5 = 5000000
+    exp_diff10 = 10000000
+    child.expect('This test will print "Slept for X sec..." every 1, 5 and 10 seconds.\r\n')
```

Comment:
If you make these strings unicode (prefixing with `U`) the script should be Python 2 and Python 3 compatible.